File: 05_이원영/1110.js
//4-3 1110
var input = 26;
var num = input;
var cycle = 0;

do {
    cycle++;
    var a = parseInt(num / 10);
    var b = num % 10;

    num = b * 10 + ((a+b) % 10);
} while(input != num)

console.log(cycle);

File: 01_김아경/week01/1110.py
N = int(input())
n = N
count = 0

while count == 0 or n != N: #F or T = T
    count += 1
    
    if n < 10:
        n = n*10 + n
    else:
        n = n%10*10 + (n//10 + n%10)%10
    
    

print(count)